### DISCUSSION TITLE
setting `--runtime-flag` via `containers.conf`

### DISCUSSION THREAD

**Discussion by felschr** [containers/podman#20228] [Q&A] [answered]
Is it possible to set `--runtime-flag` via the `containers.conf`?

It has been asked in https://github.com/containers/podman/issues/6986, but the PR that addresses this only seems to have added `--runtime-flag` itself, but no option for `containers.conf`.

**Comment by vrothberg** [comment]
Thanks for reaching out.  `containers.conf` has a `runtime` flag in the `[Engine]` table (see https://github.com/containers/common/blob/main/docs/containers.conf.5.md) which should do what you are looking for.

**Reply by felschr** [reply]
That's just an option to specify the runtime itself. I'm looking for an option to set extra flags/arguments that get passed to the runtime.

**Reply by vrothberg** [reply, marked as answer]
My apologies, I should have read more carefully.  You are right, at the moment there is no counterpart to `--runtime-flag` in containers.conf.

**Reply by vrothberg** [reply]
Would you create an issue/RFC for it?

**Reply by dseomn** [reply]
I was looking for a way to specify runtime flags in containers.conf and found this. I didn't see any issue created, so I made #25565.
